app/models/agents/liquid_output_agent.rb

```diff
@@ -0,0 +1,199 @@
+module Agents
+  class LiquidOutputAgent < Agent
+    include WebRequestConcern
+    include FormConfigurable
+
+    cannot_be_scheduled!
+    cannot_create_events!
+
+    DATE_UNITS = %w[second seconds minute minutes hour hours day days week weeks month months year years]
+
+    description  do
+      <<-MD
+        The Liquid Output Agent outputs events through a Liquid template you provide.  Use it to create a HTML page, or a json feed, or anything else that can be rendered as a string from your stream of Huginn data. 
+
+        This Agent will output data at:
+
+        `https://#{ENV['DOMAIN']}#{Rails.application.routes.url_helpers.web_requests_path(agent_id: ':id', user_id: user_id, secret: ':secret', format: :any_extension)}`
+
+        where `:secret` is the secret specified in your options.  You can use any extension you wish.
+
+        Options:
+
+          * `secret` - A token that the requestor must provide for light-weight authentication.
+          * `expected_receive_period_in_days` - How often you expect data to be received by this Agent from other Agents.
+          * `content` - The content to display when someone requests this page.
+          * `mime_type` - The mime type to use when someone requests this page.
+          * `mode` - The behavior that determines what data is passed to the Liquid template.
+          * `event_limit` - A limit applied to the events passed to a template when in "Last X events" mode. Can be a count like "1", or an amount of time like "1 day" or "5 minutes".
+
+        # Liquid Templating
+
+        The content you provide will be run as a Liquid template. The data from the last event received will be used when processing the Liquid template.
```

Comment:
Maybe link [Liquid template](https://shopify.github.io/liquid/) to https://shopify.github.io/liquid/ ?